bin/cache.ml

```diff
@@ -52,17 +52,39 @@ let trim =
          (User_message.make
             [ Pp.textf "Freed %s" (Bytes_unit.pp trimmed_bytes) ])
 
+let size =
+  let info =
+    let doc = "Query the size of the Dune cache" in
+    let man =
+      [ `P "Output the size of the Dune cache in a human-readable form." ]
+    in
+    Cmd.info "size" ~doc ~man
+  in
+  Cmd.v info
+  @@ let+ machine_readble =
+       Arg.(
+         value & flag
+         & info [ "machine-readable" ]
+             ~doc:"Outputs size as a plain number of bytes.")
+     in
+     let size = Dune_cache.Trimmer.overhead_size () in
```

Comment:
What is the "overhead size"?

Reply:
It's how much disk space can be saved if Dune cache is deleted. (There is also a comment in the mli for this function.)

Note that the overhead size is less than the size of the Dune cache directory, because there are a lot of hardlinks pointing to the workspace.

Reply:
I've been a bit more specific in the --help now.